### ISSUE TITLE
PostgreSQL support?

### ISSUE THREAD

**Issue by achiku** [shirou/db2structs#2]
Is it ok to send pr to support PostgreSQL, or you are not planning to support PostgreSQL at all? (I'm guessing the name of the repo and project structure implies there is a plan to extend this tool to support other databases tho.)

**Comment by shirou** [comment]
Yes, supporting PostgreSQL is one of my desired. But, PR is always welcome!
